app/code/Magento/InventorySales/etc/module.xml

```diff
@@ -9,6 +9,7 @@
     <module name="Magento_InventorySales" setup_version="1.0.0">
         <sequence>
             <module name="Magento_InventoryCatalog"/>
+            <module name="Magento_CatalogInventory"/>
```

Comment:
this is wrong sequence, we no need to add CatalogInventory to any MSI module,
taking into account that MSI would be installed on top of Magento, CatalogInventory would be already installed.